SPECS/mlnx-ofa_kernel-hwe/mlnx-ofa_kernel-hwe.spec

```diff
@@ -100,18 +101,19 @@
 
 Summary:	 Infiniband HCA Driver
 Name:		 mlnx-ofa_kernel-hwe
-Version:	 24.10
-Release:	 23%{release_suffix}%{?dist}
+Version:	 25.07
+Release:	 1%{release_suffix}%{?dist}
 License:	 GPLv2
 Url:		 http://www.mellanox.com/
 Group:		 System Environment/Base
-Source0:         https://linux.mellanox.com/public/repo/mlnx_ofed/24.10-0.7.0.0/SRPMS/mlnx-ofa_kernel-24.10.tgz#/mlnx-ofa_kernel-%{_version}.tgz
-Patch0:          001-fix-module-init-for-ibt.patch
+# DOCA OFED feature sources come from the following MLNX_OFED_SRC tgz.
+# This archive contains the SRPMs for each feature and each SRPM includes the source tarball and the SPEC file.
+# https://linux.mellanox.com/public/repo/doca/3.1.0/SOURCES/mlnx_ofed/MLNX_OFED_SRC-25.07-0.9.7.0.tgz
```

Comment:
Double-checking: comment here says "25.07-0.9.7.0" but the `_release` macro is set to "25.07.0.9.7.1" (difference in the last number). Is this correct?

Reply:
OFED version defined in each component SPEC file as OFED.25.07.0.9.7.1. Hence update with same value based on SRPMS. Please refer to SPEC file inside SRPMS for confirmation.